feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/aggregate_all_not_forwarding_viable_test.go

```diff
@@ -0,0 +1,988 @@
+// -  there is packet rx on the port and process it to destination  with forwarding-viable=false.
+
+// What is forwarding viable ?
+// If set to false, the interface is not used for forwarding traffic,
+// but as long as it is up, the interface still maintains its layer-2 adjacencies and runs its configured layer-2 functions (e.g. LLDP, etc.).
+
+package aggregate_all_not_forwarding_viable_test
+
+import (
+	"bytes"
+	"context"
+	"encoding/binary"
+	"fmt"
+	"net"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/open-traffic-generator/snappi/gosnappi"
+	"github.com/openconfig/featureprofiles/internal/attrs"
+	"github.com/openconfig/featureprofiles/internal/deviations"
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/featureprofiles/internal/gribi"
+	"github.com/openconfig/featureprofiles/internal/otgutils"
+	"github.com/openconfig/gribigo/fluent"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/gnmi"
+	"github.com/openconfig/ondatra/gnmi/oc"
+	"github.com/openconfig/ondatra/gnmi/oc/ocpath"
+	"github.com/openconfig/ondatra/netutil"
+	"github.com/openconfig/ygnmi/ygnmi"
+	"github.com/openconfig/ygot/ygot"
+)
+
+const (
+	ipv4PLen          = 30
+	ipv6PLen          = 126
+	isisInstance      = "DEFAULT"
+	dutAreaAddress    = "49.0001"
+	ateAreaAddress    = "49"
+	dutSysID          = "1920.0000.2001"
+	asn               = 64501
+	acceptRoutePolicy = "PERMIT-ALL"
+	trafficPPS        = 2500000
+	srcTrafficV4      = "100.0.1.1"
+	srcTrafficV6      = "2002:db8:64:64::1"
+	dstTrafficV4      = "100.0.2.1"
+	dstTrafficV6      = "2003:db8:64:64::1"
+	v4Count           = 254
+	v6Count           = 100000000
+	lagTypeLACP       = oc.IfAggregate_AggregationType_LACP
+	ieee8023adLag     = oc.IETFInterfaces_InterfaceType_ieee8023adLag
+	ethernetCsmacd    = oc.IETFInterfaces_InterfaceType_ethernetCsmacd
+	LAG1              = "lag1"
+	LAG2              = "lag2"
+	LAG3              = "lag3"
+)
+
+type aggPortData struct {
+	dutIPv4      string
+	ateIPv4      string
+	dutIPv6      string
+	ateIPv6      string
+	ateAggName   string
+	ateAggMAC    string
+	ateISISSysID string
+	ateLagCount  uint32
+}
+
+type ipAddr struct {
+	ip     string
+	prefix uint32
+}
+
+// testArgs holds the objects needed by a test case.
+type testArgs struct {
+	dut    *ondatra.DUTDevice
+	ate    *ondatra.ATEDevice
+	top    gosnappi.Config
+	ctx    context.Context
+	client *fluent.GRIBIClient
+}
+
+var (
+	agg1 = &aggPortData{
+		dutIPv4:      "192.0.2.1",
+		ateIPv4:      "192.0.2.2",
+		dutIPv6:      "2001:db8::1",
+		ateIPv6:      "2001:db8::2",
+		ateAggName:   LAG1,
+		ateAggMAC:    "02:00:01:01:01:01",
+		ateISISSysID: "640000000002",
+		ateLagCount:  1,
+	}
+	agg2 = &aggPortData{
+		dutIPv4:      "192.0.2.5",
+		ateIPv4:      "192.0.2.6",
+		dutIPv6:      "2001:db8::5",
+		ateIPv6:      "2001:db8::6",
+		ateAggName:   LAG2,
+		ateAggMAC:    "02:00:01:01:02:01",
+		ateISISSysID: "640000000003",
+		ateLagCount:  5,
```

Comment:
can we make the script to work with less ports as well  instead of looking for 8 ports always . may be some thing like 1 for lag1 +2 lag2 +1 lag3 ?  or other way detect number of ports in binding file and distribute one to each LAG and attach leftover ports to LAG2

Reply:
Yeah we can do but Referring this to do it similar https://github.com/openconfig/featureprofiles/blob/1b16e43c73843951478440bef9aabdb25ad15dc0/feature/interface/aggregate/otg_tests/aggregate_forwarding_viable_test/README.md 

Also in future there may be some more subtest can be add here so others ports can help on this.

Reply:
@rahulsy26 , i see we have done this way before in test /feature/interface/aggregate/otg_tests/balancing_test/balancing_test.go where test designed for 9 ports but works with even 3 ports.

Reply:
@bkreddy143 done changes and now script is designed to work for atleast 4 ports